specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/examples/dedicatedHostExamples/DedicatedHost_Update_Resize.json

```diff
@@ -0,0 +1,63 @@
+{
+  "parameters": {
+    "resourceGroupName": "rgcompute",
+    "hostGroupName": "aaaaaaaaa",
+    "hostName": "aaaaaaaaaaaaaaaaaaaaa",
+    "api-version": "2023-03-01",
+    "parameters": {
+      "sku": {
+        "name": "DSv3-Type1"
+      }
+    },
+    "subscriptionId": "{subscription-id}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "location": "westus",
+        "tags": {},
+        "properties": {
+          "platformFaultDomain": 1,
+          "autoReplaceOnFailure": true,
+          "hostId": "{GUID}",
+          "virtualMachines": [
+            {
+              "id": "aaaa"
+            }
+          ],
+          "licenseType": "Windows_Server_Hybrid",
+          "provisioningTime": "2021-11-30T12:58:26.526Z",
+          "provisioningState": "aaaaaaaaaaaaaaaaaaaaaaaaa",
+          "instanceView": {
+            "assetId": "aaaaaaaaaaaaaaaa",
+            "availableCapacity": {
+              "allocatableVMs": [
+                {
+                  "vmSize": "aaaaaaaaaaaaaaaaaaaa",
+                  "count": 26
+                }
+              ]
+            },
+            "statuses": [
+              {
+                "code": "aaaaaaaaaaaaaaaaaaaaaaa",
+                "level": "Info",
+                "displayStatus": "aaaaaa",
+                "message": "a",
+                "time": "2021-11-30T12:58:26.522Z"
+              }
+            ]
+          }
+        },
```

Comment:
remove the instanceView as I believe we don't add it to the response of the PUT/PATCH dedicated host response.